[ADD] Subscriptions: Wire transfer (#10276)
odoo/documentation: content/applications/sales/subscriptions/payment_providers/wire_transfer.rst

be added afterwards, and the message updated with the appropriate bank information by clicking
:guilabel:`Reload pending message`.

Once all configuration settings are set, the :icon:`fa-eye-slash` :guilabel:`Unpublished`
smart button located at the top of the page automatically changes to :icon:`fa-globe`
:guilabel:`Published` to show that the wire transfer payment option is now visible to customers.

.. tip::
   Only keep the payment method published if payments are ready to be received. It is possible to
   keep the payment method enabled, but **not** visible to customers, until payments are ready to be
   received by setting the smart button to :icon:`fa-eye-slash` :guilabel:`Unpublished` after
   setting the :guilabel:`State` to :guilabel:`Enabled`.

.. image:: wire_transfer/wire-transfer-published.png
   :align: center
   :alt: The wire transfer card enabled and published.

Test mode
---------

The wire transfer payment method can be tested before it is accessed by customers by using *Test
Mode*. To enable Test Mode, set the :guilabel:`State` in the payment provider form to
:guilabel:`Test Mode` and an orange :guilabel:`Test mode` banner appears in the upper-right side of

> the form.

Reviewer: Do you think it's worth mentioning that when Test Mode is selected, the method becomes 'Unpublished'?

Next, enter the credentials for a bank account with accessible transactions in the
:guilabel:`Messages` tab. To re-enable normal function after testing, change :guilabel:`State` to
:guilabel:`Enabled`.

.. note::

>    By default, the payment provider remains **unpublished** in test mode so that it is not visible

Reviewer: ```suggestion
   By default, the payment provider remains :guilabel:`Unpublished` in :guilabel:`Test Mode`, so it is not visible
```

   to visitors.

.. warning::

>    It is recommended to use test mode on a duplicate or a test database to avoid potential issues

Reviewer: ```suggestion
   It is recommended to use the :guilabel:`Test Mode` setting on a duplicate (or a test) database to avoid potential issues
```

   with invoice numbering.

.. _wire_transfer/payment:

Pay by wire transfer
====================

> With wire transfers :ref:`enabled and published <wire_transfer/configuration>`, customers will see

Reviewer: ```suggestion
With wire transfers :ref:`enabled and published <wire_transfer/configuration>`, customers see
```

> wire transfers as an available payment method at checkout when purchasing a product. When this
> method is selected by a customer, they are shown the *Pending Message* set in the :ref:`wire
> transfer configuration page <wire_transfer/configuration>` after confirming their purchase. Once
> the purchase is confirmed, a sales order is created.

Reviewer: ```suggestion
wire transfers as an available payment method at checkout when purchasing a product.

When this method is selected by a customer, they are shown the *Pending Message* set in the :ref:`wire
transfer configuration page <wire_transfer/configuration>` after confirming their purchase. Once
the purchase is confirmed, a sales order is created.
```

.. image:: wire_transfer/payment-instructions-checkout.png
   :align: center
   :alt: The wire transfer pending message shown to customers.

.. important::
   Online orders remain in the *Quotation sent* (i.e. unpaid order) stage until both the payment is

>    received, *and* the order confirmed manually using the :guilabel:`Confirm` button on the sales

Reviewer: ```suggestion
   received, *and* the order is confirmed manually, using the :guilabel:`Confirm` button on the sales
```

   order.

> To begin processing the payment, navigate to :menuselection:`Sales app --> Orders --> Orders` and

Reviewer: ```suggestion
To begin processing the payment, navigate to :menuselection:`Sales app --> Orders --> Orders`, and
```

> click the corresponding sales order. A green :guilabel:`In Progress` banner appears in the

Reviewer: During my run-through, I wasn't seeing this 'In Progress' banner -- might be worth a quick double-check during your revisions.

upper-right to show that payment has not been confirmed yet. Ensure the sales order is correct, then
click :guilabel:`Confirm` to confirm the sales order.

> Next, click the new :guilabel:`Create Invoice` button, then choose :guilabel:`Regular invoice` and

Reviewer: ```suggestion
Next, click the new :guilabel:`Create Invoice` button, then choose :guilabel:`Regular invoice`, and
```

click :guilabel:`Create Draft Invoice`. A draft of the invoice appears. Click :guilabel:`Confirm` to
validate the invoice.

To send this invoice to the customer, click the :guilabel:`Send & Print` button. A :guilabel:`Send`

> pop-up window is revealed with an email template configured and a PDF invoice attached.

Reviewer: ```suggestion
pop-up window appears, with an email template configured, and a PDF invoice attached.
```

Ensure the information is correct, then click :guilabel:`Send & Print`. The pop-up window

> disappears and the :guilabel:`Send & Print` button turns gray to show the invoice has been sent.

Reviewer: ```suggestion
disappears, and the :guilabel:`Send & Print` button turns gray to show the invoice has been sent.
```

.. tip::
   The default template for sending an invoice can be modified by navigating to
   :menuselection:`Settings app --> Companies --> Email Templates --> Review All Templates -->
   Invoice: Sending`.

> Now click the :guilabel:`Register payment` button on the invoice page. A pop-up window titled

Reviewer: ```suggestion
Next, click the :guilabel:`Register payment` button on the invoice page. A pop-up window titled,
```

> :guilabel:`Register Payment` is revealed with options for selecting the type of payment. In the

Reviewer: ```suggestion
:guilabel:`Register Payment` appears, with options for selecting the type of payment. In the
```

> field to the right of :guilabel:`Journal`, type `Bank` or select it from the drop-down menu. Next,

Reviewer: ```suggestion
:guilabel:`Journal` field, select :guilabel:`Bank` from the drop-down menu. Next,
```

> set :guilabel:`Payment Method` to `Manual`. Ensure :guilabel:`Amount` and :guilabel:`Payment Date`

Reviewer: ```suggestion
set the :guilabel:`Payment Method` to :guilabel:`Manual`. Ensure the :guilabel:`Amount` and :guilabel:`Payment Date`
```

are correct, then click the :guilabel:`Create Payment` button.

.. image:: wire_transfer/register-payment.png
   :align: center
   :alt: The register payment pop-up window.

> The customer invoice is revealed with a green :guilabel:`In payment` banner in the upper-right

Reviewer: ```suggestion
The customer invoice is revealed with a green :guilabel:`In Payment` banner in the upper-right
```

corner. At this point, the flow is now complete.

.. image:: wire_transfer/in-payment-banner.png
   :align: center
   :alt: The green in payment banner displayed on the invoice.

.. note::
   If payment has been received before sending the invoice, such as in cash, the payment can still

>    be registered in Odoo by clicking the :guilabel:`Register Payment` button and creating the

Reviewer: ```suggestion
   be registered in Odoo by clicking the :guilabel:`Register Payment` button, and creating the
```

>    payment *before* clicking :guilabel:`Send & Print`. Now when an invoice is sent using the

Reviewer: ```suggestion
   payment *before* clicking :guilabel:`Send & Print`. Now, when an invoice is sent using the
```

   :guilabel:`Send & Print` button, a new line is automatically added to the invoice message

>    stating that the invoice is already paid.

Reviewer: ```suggestion
   stating the invoice is already paid.
```

   sales order.

To begin processing the payment, navigate to :menuselection:`Sales app --> Orders --> Orders`, and
click the corresponding sales order. Ensure the sales order is correct, then click
:guilabel:`Confirm` to confirm the sales order.

Next, click the new :guilabel:`Create Invoice` button, then choose :guilabel:`Regular invoice`, and
click :guilabel:`Create Draft Invoice`. A draft of the invoice appears. Click :guilabel:`Confirm` to
validate the invoice.

To send this invoice to the customer, click the :guilabel:`Send & Print` button. A :guilabel:`Send`
pop-up window appears, with an email template configured, and a PDF invoice attached.

Ensure the information is correct, then click :guilabel:`Send & Print`. The pop-up window
disappears, and the :guilabel:`Send & Print` button turns gray to show the invoice has been sent.

.. tip::
   The default template for sending an invoice can be modified by navigating to
   :menuselection:`Settings app --> Companies --> Email Templates --> Review All Templates -->
   Invoice: Sending`.

Next, click the :guilabel:`Register payment` button on the invoice page. A pop-up window titled,

> :guilabel:`Register Payment` appears, with options for selecting the type of payment. In the

Reviewer: ```suggestion
:guilabel:`Register Payment`, appears with options for selecting the type of payment. In the
```

:guilabel:`Journal` field, select :guilabel:`Bank` from the drop-down menu. Next, set the
:guilabel:`Payment Method` to :guilabel:`Manual`. Ensure the :guilabel:`Amount` and
:guilabel:`Payment Date` are correct, then click the :guilabel:`Create Payment` button.

.. image:: wire_transfer/register-payment.png
   :align: center
   :alt: The register payment pop-up window.

The customer invoice is revealed with a green :guilabel:`In Payment` banner in the upper-right
corner. At this point, the flow is now complete.

.. image:: wire_transfer/in-payment-banner.png
   :align: center
   :alt: The green in payment banner displayed on the invoice.

.. note::
   If payment has been received before sending the invoice, such as in cash, the payment can still
   be registered in Odoo by clicking the :guilabel:`Register Payment` button, and creating the
   payment *before* clicking :guilabel:`Send & Print`. Now, when an invoice is sent using the

>    :guilabel:`Send & Print` button, a new line is automatically added to the invoice message

Reviewer: ```suggestion
   :guilabel:`Send & Print` button, a new line is automatically added to the invoice message,
```
